web3/utils/datastructures.py

```diff
@@ -95,6 +99,15 @@ def __eq__(self, other):
         else:
             return False
 
+    @staticmethod
+    def serializeHexBytes(value):
+        return value.hex() if isinstance(value, HexBytes) else value
```

Comment:
This should probably be `isinstance(value, bytes)` to catch both `bytes` and `HexBytes` types.